dev-node/assert-plus/assert-plus-1.0.0.ebuild

```
@@ -0,0 +1,14 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit node
+
+DESCRIPTION="Extra assertions on top of node's assert module"
+HOMEPAGE="
+	https://github.com/mcavage/node-assert-plus
+	https://www.npmjs.com/package/assert-plus
+"
+LICENSE="MIT"
```

```suggestion
HOMEPAGE="
	https://github.com/mcavage/node-assert-plus
	https://www.npmjs.com/package/assert-plus
"

LICENSE="MIT"
```
I'd love a space here

---

Done

---

Done